rotkehlchen/chain/ethereum/oracles/uniswap.py

```diff
@@ -68,6 +68,9 @@ def __init__(self, eth_manager: 'EthereumManager', version: int):
             A_USDT.resolve_to_evm_token(),
         ]
 
+    def is_punished(self) -> bool:
```

Comment:
I know the name was to punish, but this was just a figure of speech to explain what should happen.

Please use different word. What is happening? The oracle is penalized due to having not beeng timing out or generally misbehaving. Use that word instead everywhere.